### PR TITLE
Add metric 'AIS-05-M1' for discussion.

### PR DESCRIPTION
**Pull request by apannetrat**
This partially addresses issue #23 
Two more metrics are needed.

### CODE REVIEW

**Comment by mosi-k-platt** [comment]
Point of concern: Adding a specific SAST metric to the MVP could feel like a de facto endorsement of SAST as a key security control that should be prioritized before IAST, DAST or SCA.

**Comment by apannetrat** [comment]
There are now 3 metrics in this branch.
In addition, I've added the CCM description of each primary CCM control referenced by a metric. 

The CCM controls are stored in the primary dataset as an extra table called `ccm_references` in the YAML primary dataset. This avoids duplicating control descriptions and will potentially allow adding descriptions for _related CCMv4 controls ids_.

File: data/primary-dataset.yml [hunk 1]
```diff
@@ -31,12 +31,69 @@ metrics:
...
+    - id: prod_apps_deployed
+      name: B
+      description: Total number of units of code that are deployed in production.
+  rules: |
+    **Production Code** is code deployed to the production runtime environment(s)
+    within the scope of the Information Security Program defined in the CCMv4 GRC-05
+    control objective. 
+    Testing may be applied at the unit, system, integration or functional level, or any
+    combination thereof. Testing requirements and coberage should be be pre-defined in 
+    the organization's software development policy.
```
Comment (pritikin) [inline data/primary-dataset.yml:59]: How much of rules are audit guidelines and vice versa?

File: data/primary-dataset.yml [hunk 2]
```diff
@@ -1201,3 +1201,171 @@ metrics:
     sloRangeMin: 99%
   implementationGuidelines: This depends on an Asset Database such as from CCMv4 DCS-06.  The targeted classifications of assets in scope must be identified in CCMv4 DCS-05 (e.g. employee devices).
   samplingPeriod: P30D
+ccm_references:
```
Comment (pritikin) [inline data/primary-dataset.yml:1204]: perhaps 'ccmv4_references' so we can handle future versions

Comment (pritikin) [reply data/primary-dataset.yml:1204]: alian points out "ccm_version" earlier in the file

**Review by pritikin — APPROVED** [review]

**Pull request merged**